crates/libcontainer/src/tty.rs

```diff
@@ -75,7 +76,13 @@ pub fn setup_console_socket(
     console_socket_path: &Path,
     socket_name: &str,
 ) -> Result<RawFd> {
-    let linked = container_dir.join(socket_name);
+    // Move into the container directory to avoid sun family conflicts with long socket path names.
```

Comment:
```suggestion
    // Move into the container directory to avoid sun family conflicts with long socket path names.
    // ref: https://github.com/containers/youki/issues/2910
```